string pcd_path # path to pcd with object model
---